components/User/AdminUserProfilePage.vue

```diff
@@ -278,8 +298,34 @@ const passwordId = useId()
 
 const loading = ref(false)
 
+const profilePicture = ref<File | null>(null)
+
+watchEffect(() => {
+  if (me.value.about === null) {
+    me.value.about = ''
+  }
+})
+
+const setFiles = (files: Array<File>) => {
+  profilePicture.value = files[0]
+}
+
+const imagePreview = computed(() => {
+  if (!profilePicture.value) return null
+  if (typeof profilePicture.value === 'string') return profilePicture.value
+  return URL.createObjectURL(profilePicture.value)
+})
+
 async function updateMe() {
   loading.value = true
+  if (profilePicture.value) {
+    try {
+      await uploadProfilePicture(profilePicture.value)
```

Comment:
Do we want to update the avatar at the top of the page upon upload?

Reply:
It should update both the avatar on the admin page and small one inside the header

Reply:
Indeed, it is a case after a reinstall!